### PR TITLE
Fix/UI misc improvements fixes and upgrades

### PR DESCRIPTION
**Pull request by evandrosaturnino**
[X] - Application still loads/functions properly if the tvl isn't queried for any reason (error handled and displayed in console)
[X] - Graceful error handling for lack of config.json file
[X] - Recurrent tvl fetching along time for seamless update in continuous time
[X] - Dashboard chart promise chain better structured semantically
[X] - Promise handling logic improvement for fetching blocks by timestamp (for querying the TVL)
[X] - Date label displayed in the tvl chart according to the user local time (Always current updated time for the user)
[X] - Only display chart position value on hover, otherwise it doesn't display any data
[X] - Chart data always begins on zero for better data presentation
[X] - Improved chart layout, fixing index overlap
[X] -Implemented filter to change the color of svg conditionally based on the color theme (instead of changing the svg file, improving overall loading time)
[X] -Refactored strings file to constants file (Adapted all the components to apply correct import path)
[X] -Several other minor improvements and fixes over the app (e.g. Moon and Sun icons centralized as flexbox - Removed chainid console log - Sun svg size change - Removed unused dark theme svgs)

### CODE REVIEW

File: packages/dev-frontend/src/components/Dashboard/Chart/context/ChartProvider.tsx [hunk 1]
```diff
@@ -120,34 +114,50 @@ async function fetchData(API_URL: string, query: string) {
...
+  const [ tvl, setTvl ] = useState<Array<tvlData>>();
+  const [ isMounted, setIsMounted ] = useState<boolean>(true);
   const { config, provider } = useLiquity();
   const { BlocksApiUrl, ThresholdUsdApiUrl } = config;
 
+  const getTVLData = () => {
+    return provider.getNetwork()
+    .then((network) => {
+      const networkName = network.name === 'homestead' ? 'ethereum' : network.name;
+      const BlocksUrlByNetwork = `https://${BlocksApiUrl}/${networkName}-blocks`;
```
Comment (benlongstaff) [inline packages/dev-frontend/src/components/Dashboard/Chart/context/ChartProvider.tsx]: should BlocksUrlByNetwork be blocksUrlByNetwork ?

File: packages/dev-frontend/src/components/Dashboard/Chart/context/ChartProvider.tsx [hunk 2]
```diff
@@ -120,34 +114,50 @@ async function fetchData(API_URL: string, query: string) {
...
+      const ThresholdUrlByNetwork = `https://${ThresholdUsdApiUrl}/${networkName}-thresholdusd`;
```
Comment (benlongstaff) [inline packages/dev-frontend/src/components/Dashboard/Chart/context/ChartProvider.tsx]: should ThresholdUrlByNetwork be lowercase?

File: packages/dev-frontend/src/components/Dashboard/Chart/context/ChartProvider.tsx [hunk 3]
```diff
@@ -120,34 +114,50 @@ async function fetchData(API_URL: string, query: string) {
 };
 
 export const ChartProvider: React.FC<FunctionalPanelProps> = ({ children, loader })  => {
-  const timestamps: Array<number> = createListOfTimestamps();
-  const [tvl, setTvl] = useState<Array<tvlData>>();
-  const [isMounted, setIsMounted] = useState<boolean>(true);
-
+  const timestamps: Array<TimestampsObject> = createListOfTimestamps();
+  const [ isTVLDataAvailable , SetisTVLDataAvailable ] = useState<boolean>(true);
```
Comment (benlongstaff) [inline packages/dev-frontend/src/components/Dashboard/Chart/context/ChartProvider.tsx]: should this be setIsTVLDataAvailable

**Review by benlongstaff — APPROVED** [review]

File: packages/dev-frontend/src/components/Dashboard/Chart/context/ChartProvider.tsx (hunk 3 above)
Comment (evandrosaturnino) [reply packages/dev-frontend/src/components/Dashboard/Chart/context/ChartProvider.tsx]: solved in https://github.com/Threshold-USD/dev/pull/56/commits/3c001e46a13991a21d49412d1d0c9fb0d027eed2, thanks

File: packages/dev-frontend/src/components/Dashboard/Chart/context/ChartProvider.tsx (hunk 2 above)
Comment (evandrosaturnino) [reply packages/dev-frontend/src/components/Dashboard/Chart/context/ChartProvider.tsx]: solved in https://github.com/Threshold-USD/dev/pull/56/commits/3c001e46a13991a21d49412d1d0c9fb0d027eed2, thanks

File: packages/dev-frontend/src/components/Dashboard/Chart/context/ChartProvider.tsx (hunk 1 above)
Comment (evandrosaturnino) [reply packages/dev-frontend/src/components/Dashboard/Chart/context/ChartProvider.tsx]: solved in https://github.com/Threshold-USD/dev/pull/56/commits/3c001e46a13991a21d49412d1d0c9fb0d027eed2, thanks

**Pull request merged**